At 2522 tagging endpoints (#432)
up42/up42-py: docs/reference/tasking-reference.md

| `acquisition_start` | `Union[str, datetime]`                                                        | The start date of the acquisition period in the `YYYY-MM-DD` format.                                                                          |
| `acquisition_end`   | `Union[str, datetime]`                                                        | The end date of the acquisition period in the `YYYY-MM-DD` format.                                                                            |
| `geometry`          | `Union[FeatureCollection, Feature, dict, list, GeoDataFrame, Polygon, Point]` | Geometry of the area to be captured. It can be a POI or an AOI depending on the [collection](https://docs.up42.com/data/tasking/limitations). |

> | `tags`              | `Optional[List[str]` | Add tags to order with the provided tags. |

Reviewer: ```suggestion
| `tags`              | `Optional[List[str]` | A list of tags that categorize the order. |
```
